tests/integration_tests/test_game.py

```diff
@@ -231,6 +232,147 @@ async def test_game_ended_rates_game(lobby_server):
         await read_until_command(host_proto, "player_info", timeout=10)
 
 
+#@pytest.mark.rabbitmq
```

Comment:
If you need to run this test locally use `-m rabbitmq` or `-m ""` to run all tests

Reply:
Argh. I had forgotten how to do that so I did the obvious stupid thing and of course it came back to bite me.